kong/plugins/aws-lambda/handler.lua

```diff
@@ -38,6 +38,51 @@ local server_header = meta._SERVER_TOKENS
 local AWS_PORT = 443
 
 
+--[[
+  Response format should be
+  {
+      "statusCode": httpStatusCode,
+      "headers": { "headerName": "headerValue", ... },
+      "body": "..."
+  }
+--]]
+local function validate_custom_response(response)
+  if type(response.statusCode) ~= "number" then
+    return nil, "statusCode must be a number"
+  end
+
+  local headers = response.headers
+  local body = response.body
```

Comment:
There is no point in de-referencing those values it seems. The below code even indexes `response.body` still. It seems concise enough.